cmake/kokkos_test_cxx_std.cmake

```diff
@@ -35,18 +35,9 @@ FUNCTION(kokkos_set_cxx_standard_feature standard)
     GLOBAL_SET(KOKKOS_CXX_STANDARD_FEATURE "")
   ENDIF()
 
-    IF(NOT ${FEATURE_NAME} IN_LIST CMAKE_CXX_COMPILE_FEATURES)
-          IF (KOKKOS_CXX_COMPILER_ID STREQUAL "NVIDIA")
-                  MESSAGE(STATUS "nvcc_wrapper does not support TARGET_COMPILE_FEATURES")
-                        GLOBAL_SET(KOKKOS_CXX_STANDARD_FEATURE "")
-                            ELSE()
-                                    MESSAGE(FATAL_ERROR "Compiler ${KOKKOS_CXX_COMPILER_ID} should support ${FEATURE_NAME}, but CMake reports feature not supported")
-                                        ENDIF()
-                                          ENDIF()
-
-  #  IF(NOT ${FEATURE_NAME} IN_LIST CMAKE_CXX_COMPILE_FEATURES)
-  #    MESSAGE(FATAL_ERROR "Compiler ${KOKKOS_CXX_COMPILER_ID} should support ${FEATURE_NAME}, but CMake reports feature not supported")
-  #  ENDIF()
+  IF(NOT ${FEATURE_NAME} IN_LIST CMAKE_CXX_COMPILE_FEATURES)
+    MESSAGE(FATAL_ERROR "Compiler ${KOKKOS_CXX_COMPILER_ID} should support ${FEATURE_NAME}, but CMake reports feature not supported")
+  ENDIF()
```

Comment:
You are setting yourself up for conflicts when we will merge into develop.  You should not be editing that file at all.

Reply:
I reverted a change which Jeremiah said shouldn't have happened in the first place. This is not a conflict.